README.md

```diff
@@ -566,6 +567,24 @@ P.S. Look at [testify's issue](https://github.com/stretchr/testify/issues/772),
 
 ---
 
+### http-const
+
+```go
+❌
+assert.HTTPStatusCode(t, handler, "GET", "/index", nil, 200)
+assert.HTTPBodyContains(t, handler, "GET", "/index", nil, "counter")
+
+✅
+assert.HTTPStatusCode(t, handler, http.MethodGet, "/index", nil, http.StatusOK)
+assert.HTTPBodyContains(t, handler, http.MethodGet, "/index", nil, "counter")
+```
+
+**Autofix**: true. <br>
+**Enabled by default**: true. <br>
+**Reason**: Is similar to the [usestdlibvars](https://golangci-lint.run/usage/linters/#usestdlibvars) linter. <br>
```

Comment:
This one is for @Antonboom, Matthieu copy-pasted, and so moved the section available in Contributions.md

While the Reason was OK for a Dev purpose, I would expect a few more information by being a random Gopher finding this line. people are reading fast, and laziness/business might lead to do not click on the link.

So maybe if you could think about a better idea

At least something like that it doesn't have to be more

```suggestion
**Reason**: Cleaner code, this ckecker is similar to the [usestdlibvars](https://golangci-lint.run/usage/linters/#usestdlibvars) linter. <br>
```